drivers/sensor/bosch/bma4xx/bma4xx.c

```diff
@@ -34,19 +34,19 @@ static int bma4xx_offset_to_reg_val(const struct sensor_value *val, uint8_t *reg
 /**
  * @brief Set the X, Y, or Z axis offsets.
  */
-static int bma4xx_attr_set_offset(const struct device *dev, enum sensor_channel chan,
+static int bma4xx_attr_set_offset(const struct device *dev, enum sensor_channel ch,
```

Comment:
Should this be:
```suggestion
static int bma4xx_attr_set_offset(const struct device *dev, struct sensor_chan_spec ch,
```
?

Reply:
Yes it should be, thank you

Reply:
In looking at this closer it seems like its correct as it is, the set_offset function here is being called by bma4xx_attr_set. I didn't update the sensor_attr_get/sensor_attr_set functions to take a sensor_chan_spec as this would be a breaking change.

Reply:
Agreed, change shouldn't have made it in, removed this change from the PR